dotnet/hello_world/HelloWorldSettings.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace HelloWorld
```

Comment:
Standard Google copyright header with Apache license at the top of every *.cs file, please.

You can generate it with https://github.com/mbrukman/autogen as follows:

```bash
autogen foo.cs
```